src/constants/networks.ts

```diff
@@ -76,7 +76,9 @@ export const MAINNET_NETWORKS = [
   ChainId.VELAS,
   ChainId.AURORA,
   ChainId.ETHW,
+  ChainId.GÖRLI, // todo danh
```

Comment:
Just remove it. You dont need to add it btw. You can switch to gorli on metamask anyway